scripts/release.py

```diff
@@ -46,7 +47,7 @@ def main():
     jar = build_binary()
 
     # release
-    release_command = ['hub', 'release', 'create', '-a', jar, '-m', title, tag_name]
+    release_command = ['hub', 'release', 'create', '-o', '-a', jar, '-m', title, tag_name]
```

Comment:
For scripts, it is better to use long flags than short flags. 

Short flags are useful for interactive shells, while long flags are helpful for documentation purpose.